developer_manual/core/unit-testing.rst

```diff
@@ -12,7 +12,13 @@ Getting PHPUnit
 ownCloud >= 10.0
 ^^^^^^^^^^^^^^^^
 
-If you are using ownCloud 10.0 or higher, running ``make test-php`` or ``make`` in your terminal from the root directory will install a local version of PHPUnit inside of ``lib/composer`` and run the test suite.
+If you are using ownCloud 10.0 or higher, running ``make`` in your terminal from the ``webroot`` directory will prepare everything for testing. 
+This will install beside necessary dependencies, a local version of PHPUnit at ``<webroot>/lib/composer/phpunit/phpunit``.
```

Comment:
This shouldn't be needed in regular setups. A developer should not need to know the full path to the internal phpunit. `make test-php` takes care of that.

It only happened on @mmattel's setup that the execute perm was missing because he is using the git version together with the "hardening" guidelines. This is not a supported use case.

But if you do want to add this somewhere, maybe it would belong in a troubleshooting section.